test/modules/survey/components/Survey.test.js

```diff
@@ -0,0 +1,163 @@
+import { Survey, SURVEY_NOTIFICATION_DELAY_TIME } from '../../../../src/modules/survey/components/Survey';
+import { TestUtils } from '../../../test-utils.js';
+import { $injector } from '../../../../src/injection';
+import { createNoInitialStateMainMenuReducer } from '../../../../src/store/mainMenu/mainMenu.reducer';
+import { createNoInitialStateMediaReducer } from '../../../../src/store/media/media.reducer';
+import { notificationReducer } from '../../../../src/store/notifications/notifications.reducer';
+import { isTemplateResult } from '../../../../src/utils/checks';
+import { render } from 'lit-html';
+
+
+window.customElements.define(Survey.tag, Survey);
+
+
+describe('Survey', () => {
+	let store;
+	const setup = (state = {}, config = {}) => {
+		const { embed = false } = config;
+
+		const initialState = {
+			mainMenu: {
+				open: true
+			},
+			notifications: {
+				latest: null
+			},
+			media: {
+				portrait: false,
+				minWidth: true
+			},
+			...state
+		};
+
+		store = TestUtils.setupStoreAndDi(initialState, { mainMenu: createNoInitialStateMainMenuReducer(), media: createNoInitialStateMediaReducer(), notifications: notificationReducer });
+		$injector.registerSingleton('EnvironmentService', {
+			isEmbedded: () => embed
+		});
+		$injector.registerSingleton('TranslationService', { translate: (key) => key });
+
+		return TestUtils.render(Survey.tag);
+	};
+
+	describe('when initialized', () => {
+		beforeEach(async () => {
+			jasmine.clock().install();
+		});
+
+		afterEach(function () {
+			jasmine.clock().uninstall();
+		});
+
+		it('adds survey for landscape move', async () => {
```

Comment:
more precise would be something like 'renders  for landscape mode'

let's combine it with `describe('responsive layout '` in line 141